inference-engine/src/readers/ir_reader/ie_ir_version.hpp

```diff
@@ -22,7 +22,11 @@ inline size_t GetIRVersion(pugi::xml_node& root) {
 size_t GetIRVersion(std::istream& model) {
     std::array<char, 512> header = {};
 
+    model.seekg(0, model.end);
+    auto len = model.tellg();
     model.seekg(0, model.beg);
+    if (len < 512)
+        return false;
```

Comment:
why do we need it? If file size < 512, I suppose everything should be OK:
```
istream& read (char* s, streamsize n);
Read block of data
Extracts n characters from the stream and stores them in the array pointed to by s.

This function simply copies a block of data, without checking its contents nor appending a null character at the end.

If the input sequence runs out of characters to extract (i.e., the end-of-file is reached) before n characters have been successfully read, the array pointed to by s contains all the characters read until that point,
```

Reply:
Yes I forgot to remove this change.

Reply:
Fixed